utils/types.ts

```diff
@@ -13,3 +14,23 @@ export function assertNever(
 ): never {
   throw new Error(message);
 }
+
+/**
+ * Parses a string into a deserialized json
+ */
+export const jsonFromString = new t.Type<object, string>(
+  "JSONFromString",
+  t.object.is,
+  (m, c) => {
+    return t.string.validate(m, c).chain(s => {
+      try {
+        const prova = JSON.parse(s);
```

Comment:
```suggestion
        const test = JSON.parse(s);
```
:innocent:

Reply:
Upsi.. :sweat_smile: 

Fixed in [db944b4](https://github.com/pagopa/io-functions-pushnotifications/pull/24/commits/db944b4f39afabacaaf1f975bc44fb1704da2709)